<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_null_sensorDialog</class>
 <widget class="QDialog" name="dialog_null_sensorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_sensorDialog_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>传感器</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_sensorDialog_shutdown">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>0</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_sensorDialog_sensor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>31</y>
     <width>831</width>
     <height>492</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_sensor_general">
    <attribute name="title">
     <string>常规</string>
    </attribute>
    <widget class="QLabel" name="label_general_sensor">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>60</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>传感器</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_general_sensorName">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>80</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>传感器名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_general_sensorDataLow">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>最低值</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_general_sensorDataHigh">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>最高值</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_general_unit">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>50</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>单位</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_general_list">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>71</y>
       <width>480</width>
       <height>240</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_general_use">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>350</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_general_sure">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>350</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_general_cancel">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>350</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>类型</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_sensor_warn">
    <attribute name="title">
     <string>报警</string>
    </attribute>
    <widget class="QLabel" name="label_warn_sersor">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>传感器</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_warn_lowWarnData">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>低位报警值</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_warn_highWarnData">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>高位报警值</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_warn_Out">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>报警输出</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_warn_list">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>500</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_warn_use">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>350</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_warn_sure">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>350</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_warn_cancel">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>350</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>类型</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
